packages/sirius-web/backend/sirius-web-services/src/main/java/org/eclipse/sirius/web/services/documents/UploadDocumentEventHandler.java

```diff
@@ -160,6 +163,17 @@ public void handle(One<IPayload> payloadSink, Many<ChangeDescription> changeDesc
         changeDescriptionSink.tryEmitNext(changeDescription);
     }
 
+    private String getReport(JsonResource resource) {
```

Comment:
Why `JsonResource`? Shouldn't the reporting be done on the uploaded resource and not on the transformed resource, for example a XMI before its transformation into a JSON resource? Let's say someone uploads a XText resource with syntax warnings that would be lost in the XText to JSON transformation, shouldn't specifiers have the ability to report that information? Some things will be lost in the translation.

Reply:
It is just a mistake, I will change it.